Microsoft/WebTemplateStudio
# Dropdown different label and value

This PR fixes #552. Dropdowns on function modal can have different labels and values now, allowing us to use a different internal and displayed name for dropdowns.

_**Testing Notes:**_

- Launch the extension in VSCode
- Navigate to Services page and add Azure Functions. You'll see that runtime stack shows *JavaScript* instead of *node* now. The internal name used is still *node* since Azure ARM uses *node* internally.
- Make sure everything else is still working regularly
